<ui version="4.0" >
 <class>periodic_report_form</class>
 <widget class="QWidget" name="periodic_report_form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Periodic reports</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="lCategory" >
     <property name="text" >
      <string>Category IO :</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2" >
    <widget class="QLineEdit" name="lECategory" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4" >
    <widget class="QToolButton" name="tbCategory" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QLabel" name="lInsertDB" >
     <property name="text" >
      <string>Inserted into DB :</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3" >
    <widget class="QDateTimeEdit" name="dTEInserted" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QLabel" name="lAddressee" >
     <property name="text" >
      <string>Addressee :</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2" >
    <widget class="QLineEdit" name="lEAddressee" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4" >
    <widget class="QToolButton" name="tbAddressee" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QLabel" name="lExecutor" >
     <property name="text" >
      <string>Executor :</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2" >
    <widget class="QLineEdit" name="lEExecutor" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="4" >
    <widget class="QToolButton" name="tbExecutor" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QLabel" name="lController" >
     <property name="text" >
      <string>Controller :</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2" >
    <widget class="QLineEdit" name="lEController" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="4" >
    <widget class="QToolButton" name="tbController" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="QLabel" name="lDeadline" >
     <property name="text" >
      <string>Deadline :</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="3" >
    <widget class="QDateTimeEdit" name="dTEDeadline" />
   </item>
   <item row="6" column="0" >
    <widget class="QCheckBox" name="cbPeriod" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="1" >
    <widget class="QLabel" name="lPeriod" >
     <property name="text" >
      <string>Period :</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="2" >
    <widget class="QLineEdit" name="lEPeriod" >
     <property name="text" >
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="2" >
    <widget class="QComboBox" name="cbUnits" />
   </item>
   <item row="7" column="0" colspan="2" >
    <widget class="QLabel" name="lReason" >
     <property name="text" >
      <string>Reason :</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="3" >
    <widget class="QTextEdit" name="tEReason" />
   </item>
   <item row="8" column="1" colspan="4" >
    <widget class="QCheckBox" name="cbArchive" >
     <property name="layoutDirection" >
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text" >
      <string>Archived</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" >
    <widget class="QLabel" name="lState" >
     <property name="text" >
      <string>State :</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="2" colspan="2" >
    <widget class="QLineEdit" name="lEState" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="4" >
    <widget class="QToolButton" name="tbState" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="5" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>338</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="lReason">
     <property name="text">
      <string>Reason :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
